src/main/java/cloud/fogbow/as/core/systemidp/plugins/idpemulator/IdpEmulatorSystemIdentityProviderPlugin.java

```diff
@@ -0,0 +1,46 @@
+package cloud.fogbow.as.core.systemidp.plugins.idpemulator;
+
+import cloud.fogbow.as.constants.ConfigurationPropertyKeys;
+import cloud.fogbow.as.core.PropertiesHolder;
+import cloud.fogbow.as.core.systemidp.SystemIdentityProviderPlugin;
+import cloud.fogbow.common.constants.Messages;
+import cloud.fogbow.common.exceptions.FogbowException;
+import cloud.fogbow.common.exceptions.InvalidParameterException;
+import cloud.fogbow.common.exceptions.UnauthenticatedUserException;
+import cloud.fogbow.common.models.SystemUser;
+import org.apache.log4j.Logger;
+
+import java.util.Map;
+
+public class IdpEmulatorSystemIdentityProviderPlugin implements SystemIdentityProviderPlugin<SystemUser> {
+
+    private final Logger LOGGER;
+
+    private final String USER_NAME_FIELD = "username";
+    private final String identityProviderId;
+
+    public IdpEmulatorSystemIdentityProviderPlugin() {
+        this.LOGGER = Logger.getLogger(IdpEmulatorSystemIdentityProviderPlugin.class);
```

Comment:
to keep pattern I think it would be better if you do this in the variable declaration statement